proto/api/v2/virtual_machine_v2_service.proto

```
@@ -0,0 +1,328 @@
+syntax = "proto3";
+
+package v2;
+
+import "api/v2/scan_component.proto";
```

Buf (1.66.1)

[error] import "api/v2/scan_component.proto": file does not exist

(COMPILE)